src/containers/search_result.jsx

```diff
@@ -308,7 +317,24 @@ class SearchResult extends Component {
         ...options,
       };
 
+      const { search: { query: { sortBy } } } = this.props;
+
       const { key, label, forceShow } = opts;
+      const SortButtonSetting = {
+        highlight: false,
+        icon: {},
+      };
+      if (sortBy.key === key) {
+        SortButtonSetting.highlight = true;
+        if (sortBy.order === 'ascend') {
+          SortButtonSetting.icon = ArrowDropDown;
+        } else {
+          SortButtonSetting.icon = ArrowDropUp;
+        }
```

Comment:
You can use the tertiary operator to simplify the implementation like this:
```
SortButtonSetting.icon = sortBy.order === 'ascend' ? ArrowDropDown : ArrowDropUp;
```

Also, the 'ascend' can be put into `constants/index.js` as a common constant.

Reply:
Finished